unst_msh_gen/README.md

```diff
@@ -0,0 +1,42 @@
+# Project Name
+Unstructured mesh generation for WW3 using JIGSSAW.
+## Description
+Mesh generation script capable of creating unstructured meshes for WW3 global modeling. The tool leverages JIGSAWPY (https://github.com/dengwirda/jigsaw-python) for efficient triangulation.
+Main changes include:
+Implementation of the ocn_ww3.py to create uniform unstructured mesh for global WW3 model.
+This tool is under active development, with future work focused on variable unstructured mesh generation.
+
+## Installation
+
+1- Install jigsawpy (https://github.com/dengwirda/jigsaw-python)
+2- you need following packages:
+	numpy
+        scipy
+	packaging
+	netcdf4
+3- get the DEM:
+	$wget https://github.com/dengwirda/dem/releases/download/v0.1.1/RTopo_2_0_4_GEBCO_v2023_60sec_pixel.zip
+4- unzip the DEM 
+	unzip *.zip
+ 
+
+
+## Usage
+1- clone the repo
+	$mkdir meshgen
+	$cd meshgen
+	$git clone https://github.com/AliS-Noaa/WW3-tools/tree/add-unst-mesh-gen
```

Comment:
@AliS-Noaa this is not a valid git clone command.  Since this will be in the main branch of ww3-tools, you can either temporarily update this to check out your branch or go for the correct instructions once this is merged which would be to simply clone https://github.com/NOAA-EMC/WW3-tools